cidemipyme-gatsby/src/components/sections/ContactSection.jsx

```diff
@@ -41,22 +44,44 @@ class ContactSection extends React.Component {
     this.setState({
       [`${inputType}Text`]: event.target.value,
       showSentMessage: false,
+      showSentErrorMessage: false,
     });
   }
 
   onSubmitMessage(event) {
+    event.preventDefault();
     if (this.state.emailText.length === 0
       || this.state.subjectText.length === 0
       || this.state.messageText.length === 0
     ) {
       this.setState({
         showErrors: true
       });
-      event.preventDefault();
     } else {
-      this.setState({
-        showSentMessage: true,
-        showErrors: false
+      window.$ = window.jQuery=jquery;
+      const that = this;
+
+      $.post(
+        this.props.data.url,
+        JSON.stringify({
+          subject: this.state.subjectText,
+          email: this.state.emailText,
+          message: this.state.messageText
+        })
+      ).done(function(data) {
```

Comment:
if you use arrow functions you do not need to do this: `const that = this;`. Instead do:
```javascript
.done((data) => {
})
```